source/content/guides/load-testing-with-blazemeter.md

```diff
@@ -6,9 +6,16 @@ tags: [launch, measure, newrelic]
 type: guide
 permalink: docs/guides/:basename
 contributors: [jessifischer]
-date: 2/25/2015
+date: 3/15/2022
 ---
-Your site is loaded with content and is almost ready to go live. You know what it can do, but how will it perform in the wild? Load testing will expose slow page loads, heavy transactions, PHP errors, and database errors. [BlazeMeter](https://www.blazemeter.com/) is one of the services we recommend to use to run load tests. This guide will explain why to load test, show you how to do it, and give advice for recognizing and resolving errors you may experience.
+
+<Alert title="Warning" type="danger" >
+
+Pantheon does not recommend that you perform your own load and performance testing. We encourage you to take advantage of the load testing services offered by our [Onboarding team](/guides/professional-services/onboarding#pre-launch-load-testing). If you choose to perform your own load testing, keep in mind that traffic generated by your load testing counts as regular site traffic. See [Traffic Limits and Overages](/traffic-limits#traffic-limits-and-overages) for more information. Publishing the results of your load and performance testing is against Pantheon's [Acceptable Use Policy](https://legal.pantheon.io/).
```

Comment:
heads up to copy reviewer: that `load testing services offered by our Onboarding team` is a single pre-launch load test